src/components/Textarea/Textarea.test.tsx

```diff
@@ -40,4 +40,13 @@ describe("Textarea ", () => {
     render(<Textarea help={help} />);
     expect(screen.getByRole("textbox")).toHaveAccessibleDescription(help);
   });
+
+  it("can change value to an empty string", () => {
+    const { rerender } = render(<Textarea value="test" />);
+    const textarea = screen.getByRole("textbox");
+    expect(textarea).toHaveValue("test");
+
+    rerender(<Textarea value="" />);
+    expect(textarea).toHaveValue("");
```

Comment:
Should we make this even closer to the reality of a React state change by doing a variable change? Something like this:
```suggestion
    let testValue = "test";
    const { rerender } = render(<Textarea value={testValue} />);
    const textarea = screen.getByRole("textbox");
    expect(textarea).toHaveValue("test");

    testValue = ""; // simulate an external change of the value
    rerender(<Textarea value={testValue} />);
    expect(textarea).toHaveValue("");
```

Reply:
Makes sense, thanks for checking!